resources/views/form/login.blade.php

```diff
@@ -0,0 +1,55 @@
+@extends('layout')
+
+@section('content')
+  <div class="container">
+      <form action="/login" method="POST">
+          @csrf
+          <div class="form-group">
+              <label for="exampleInputEmail1">Email address</label>
+              <input
+                  name="email"
+                  type="email"
+                  class="form-control"
+                  id="exampleInputEmail1"
+                  aria-describedby="emailHelp"
+                  placeholder="Enter email"
+                  required
+              />
+              <small id="emailHelp" class="form-text text-muted">
+                We'll never share your email with anyone else.
+              </small>
+          </div>
+          <div class="form-group">
+              <label for="exampleInputPassword1">Password</label>
+              <input
+                  name="password"
+                  type="password"
+                  class="form-control"
+                  id="exampleInputPassword1"
+                  placeholder="Password"
+                  required
+              />
+          </div>
+          <div class="form-group form-check">
+              <input
+                  name="remember"
+                  type="checkbox"
+                  class="form-check-input"
+                  id="exampleCheck1"
+              />
+              <label class="form-check-label" for="exampleCheck1">
+                Remember me
+              </label>
```

Comment:
```suggestion
              <label class="form-check-label" for="exampleCheck1">
                  Remember me
              </label>
```
có mấy chỗ indent đang bị lệch bn check lại cho m nha